firedrake/slate/slac/utils.py

```diff
@@ -347,12 +310,9 @@ def merge_loopy(slate_loopy, builder, var2terminal):
     slate_wrapper = lp.make_function(domains, insns, args, name="slate_wrapper",
                                      seq_dependencies=True, target=lp.CTarget())
 
-    # Generate program from kernel, so that one can register and inline kernels
+    # Generate program from kernel, so that one can register kernels
     prg = make_program(slate_wrapper)
     for tsfc_loopy in tsfc_kernels:
         prg = register_callable_kernel(prg, tsfc_loopy)
-        prg = inline_callable_kernel(prg, tsfc_loopy.name)
```

Comment:
Okay, we might need to inline at some point as soon as I got vectorisation working, but maybe we want to control that from PyOP2 level anyways.